pkgs/development/libraries/CGAL/default.nix

```
@@ -13,7 +13,7 @@ stdenv.mkDerivation rec {
 
   # note: optional component libCGAL_ImageIO would need zlib and opengl;
   #   there are also libCGAL_Qt{3,4} omitted ATM
-  buildInputs = [ boost gmp mpfr ];
+  propagatedBuildInputs = [ boost gmp mpfr ];
```

we should not be propagating build inputs, unless they are required by a .pc
```suggestion
  buildInputs = [ boost gmp mpfr ];
```

---

what's .pc?

---

an output of `pkg-config` it lists what libraries need to be used by the package or propagated to transitive packages